drop-boxes/register-convert-ms-vendor-format/README.md

```diff
@@ -0,0 +1,51 @@
+# Mass Spectrometry Dropbox
+
+## Expected data structure
+In every use case, the data structure needs to contain a top folder around the respective data in order to accommodate metadata files.
+
+The sample code found in the top folder can be of type `Q_TEST_SAMPLE` or `Q_MS_RUN`. In the former case, a new sample of type `Q_MS_RUN` is created and attached as child to the test sample.
+
+**Valid folder/file types**:
+- Thermo Fisher Raw file format
+- Waters Raw folder
+- Bruker .d folder
+
+**Incoming structure overview for standard case without additional metadata file:**
+```
+QABCD102A5_20201229145526_20201014_CO_0976StSi_R05_.raw
```

Comment:
why does the top folder name has a file ending?

Reply:
good point, this is confusing. it happens because the datahandler automatically creates a folder from the file (or folder) name of the incoming data and forwards it to this and other dropboxes.
this script then expects that the main data has the same name as the top folder, so this information is important.

Reply:
I see, thanks for the explanation